GalaxyZooTouchTable/GalaxyZooTouchTable/MainWindow.xaml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using GalaxyZooTouchTable.Models;
+using System;
+using System.Windows;
+
+namespace GalaxyZooTouchTable
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+            SetUserContext();
+        }
+
+        private void SetUserContext()
+        {        
+           TableUser personUser = TableUserFactory.Create(UserType.Person);
+           TableUser lightUser = TableUserFactory.Create(UserType.Light);
+           TableUser starUser = TableUserFactory.Create(UserType.Star);
+           TableUser heartUser = TableUserFactory.Create(UserType.Heart);
+           TableUser faceUser = TableUserFactory.Create(UserType.Face);
+           TableUser earthUser = TableUserFactory.Create(UserType.Earth);
+
+           PersonUser.DataContext = personUser;
```

Comment:
For my sake, could you explain what is going here?

Reply:
The app starts with several components already initialized (`UserConsole`). Once the app opens, the `MainWindow` component is responsible for creating several user instances (via the factory) and attaching them to the already rendered `UserConsoles`. Adding the TableUsers to the DataContext of each component allows their properties to be accessible through data binding.